src/test/java/io/vertx/core/json/jackson/HybridJacksonPoolTest.java

```diff
@@ -0,0 +1,68 @@
+package io.vertx.core.json.jackson;
+
+import java.util.concurrent.CountDownLatch;
+
+import com.fasterxml.jackson.core.util.BufferRecycler;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertSame;
+import static org.junit.Assert.assertTrue;
+
+public class HybridJacksonPoolTest {
+
+  @Test
+  public void testVirtualThreadPoolWithSingleThread() {
+    HybridJacksonPool.StripedLockFreePool virtualPool = new HybridJacksonPool.StripedLockFreePool(4);
+    BufferRecycler pooledResource = virtualPool.acquirePooled();
+    assertEquals(0, virtualPool.size());
+    virtualPool.releasePooled(pooledResource);
+    assertEquals(1, virtualPool.size());
+
+    // The same thread should get the same pooled resource
+    assertSame(pooledResource, virtualPool.acquirePooled());
+    assertEquals(0, virtualPool.size());
+  }
+
+  @Test
+  public void testVirtualThreadPoolWithMultipleThreads() {
+    int stripesCount = 4;
+    HybridJacksonPool.StripedLockFreePool virtualPool = new HybridJacksonPool.StripedLockFreePool(stripesCount);
+    int nThreads = 100;
+    BufferRecycler[] resources = new BufferRecycler[nThreads];
+    CountDownLatch latch = new CountDownLatch(nThreads);
+
+    for (int i = 0; i < nThreads; i++) {
+      int threadIndex = i;
+      new Thread(() -> {
```

Comment:
This cat kill the CI, let's find an alternative way to achieve the same (expose a method which use the Id and always hit the striped variant?)

Reply:
Or assume virtual threads are available and use them via reflection?